cmd/snap/cmd_snap_op_test.go

```diff
@@ -419,6 +419,28 @@ func (s *SnapOpSuite) TestInstallDevMode(c *check.C) {
 	c.Check(s.srv.n, check.Equals, s.srv.total)
 }
 
+func (s *SnapOpSuite) TestInstallQuotaGroup(c *check.C) {
```

Comment:
we also want a TestInstallPath* variant as we have for other options

Reply:
Fixed! Thank you these notes, greatly appreciated!